testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveViewsLegacy.java

```diff
@@ -101,6 +101,37 @@ public void testHiveViewWithParametrizedTypes()
                 row("varchar(20)"));
     }
 
+    @Override
+    @Test(groups = HIVE_VIEWS)
+    public void testHiveViewWithTextualTypes()
```

Comment:
Why overridden?
if only for `SELECT column_name, data_type FROM information_schema.columns`  assertions, this can be done in a more concise manner, ensuring the bulk of the test stays in sync (because it's not duplicated)